entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+source .env
+# if lnd enabled, attempt to connect
+if [ "${LND_ENABLED}" = "1" ]
+then ./bin/agora --directory files --http-port $AGORA_PORT --lnd-rpc-authority $LND_RPC_AUTHORITY --lnd-rpc-cert-path .lnd/tls.cert --lnd-rpc-macaroon-path .lnd/invoices.macaroon
```

Comment:
You could probably simplify the entrypoint if you specify environment variables as fallbacks in arguments.rs:
https://docs.rs/structopt/0.3.25/structopt/#environment-variable-fallback

https://github.com/agora-org/agora/blob/9e8b31ae75107492a5f2494ee80b2ddc30abf9c9/src/arguments.rs#L47-L63

Reply:
Is that better? You would have the same conditional logic, but it would be in the `agora` binary instead of outside of it. It have the downside of making it less clear how the `agora` binary is configured, since if you don't see `--lnd-rpc-authority` on the command line, it might be pulling it from the environment.

Reply:
I will simplify this entrypoint, but i don't think it needs a change to the src.

Reply:
Yeah I guess it depends where you prefer to have that logic.

Reply:
I removed the `LND_ENABLED` var and used whether or not `LND_RPC_AUTHORITY` is set to determine lnd on/off